batchlib/workflows/cell_analysis.py

```diff
@@ -104,38 +105,58 @@ def parse_background_parameters(config, marker_ana_in_key, serum_ana_in_keys):
 
 def get_infected_detection_jobs(config, marker_ana_in_key, feature_identifier):
     erosion_radius = config.infected_erosion_radius
+    tophat_radius = config.infected_tophat_radius
     quantile = config.infected_quantile
 
     jobs = []
+    if tophat_radius > 0:
+        raise NotImplementedError
```

Comment:
Just a thought, do we need the background for the initial marker?
Otherwise we can just put the TopHat in front of it and update the `marker_ana_in_key`.

Reply:
You are right I think. We really only use the marker for infected cell detection, so we can do what you propose.
I implemented a similar solution now, but it is overcomplicating things. I will do what you proposed.

Reply:
I will do this properly tomorrow.. for now there is an additional variable `marker_key_for_infected_classification` that is used throughout the analysis